### PR TITLE
Document journald tag as SYSLOG_IDENTIFIER

### LINKED ISSUE #5843: Document journald tag as SYSLOG_IDENTIFIER
This issue refers to [moby/moby#35570](https://github.com/moby/moby/pull/35570).

The `journald` `tag` was not showing in basic `journalctl` output and all logs were shown as `dockerd` messages. The logging driver option `tag` was added to `SYSLOG_IDENTIFIER` (in addition to the existing `CONTAINER_TAG`). 

In this example, `journalctl` prints tag (foo), instead of unit name (dockerd) as the message source: 

`docker run --log-driver=journald --log-opt tag=foo`

### PR DESCRIPTION
**Pull request by zemanlx**
### Proposed changes

Added `SYSLOG_IDENTIFIER` to journald tag option to reflect changes in https://github.com/moby/moby/pull/35570.

Fixed linting issues 
- long lines
- table formatting
- forgotten `+` from file diff :wink:

### Related issues (optional)

Closes #5843.

### CODE REVIEW

File: config/containers/logging/journald.md [hunk 1]
```diff
@@ -46,21 +46,24 @@ To configure the logging driver for a specific container, use the `--log-driver`
 flag on the `docker run` command.
 
 ```bash
-$ docker run --log-driver=journald ...
+docker run --log-driver=journald ...
```
Comment (thaJeztah) [inline config/containers/logging/journald.md]: Can you revert this change (here and others below?) the convention used currently in the docs is to use `$` as prompt in the shell examples 😅

File: config/containers/logging/journald.md [hunk 2]
```diff
@@ -15,22 +15,22 @@ Log entries can be retrieved using the `journalctl` command, through use of the
...
+| `CONTAINER_PARTIAL_MESSAGE`          | A field that flags log integrity. Improve logging of long log lines.                                                                                   |
 
 ## Usage
 
 To use the `journald` driver as the default logging driver, set the `log-driver`
 and `log-opt` keys to appropriate values in the `daemon.json` file, which is
 located in `/etc/docker/` on Linux hosts or
 `C:\ProgramData\docker\config\daemon.json` on Windows Server. For more about
-+configuring Docker using `daemon.json`, see
-+[daemon.json](/engine/reference/commandline/dockerd.md#daemon-configuration-file).
```
Comment (thaJeztah) [inline config/containers/logging/journald.md]: Ah, nice catch; looks like a copy/paste gone wrong 😃

File: config/containers/logging/journald.md (hunk 1 above)
Comment (zemanlx) [reply config/containers/logging/journald.md]: Sure, my linter raises it as an issue [no-shell-dollars](https://github.com/remarkjs/remark-lint/tree/master/packages/remark-lint-no-shell-dollars#example) ... and I found it useful when copying the whole paragraph not to elaborate with removing `$` form each line :wink:. Maybe it is worth to consider this in future.

**Comment by gbarr01** [comment]
@thaJeztah , do you still have requested changes?
